devtools/server/actors/replay/connection.js

```diff
@@ -185,7 +185,7 @@ class Recording extends EventEmitter {
 
   async _onFinished(data) {
     try {
-      this.emit("finished");
+      this.emit("finished", data);
 
       // If we locked the recording because of sourcemaps, we should wait
```

Comment:
Is this comment still relevant? Does the risk still exist?

Reply:
I think it is still relevant because I think it refers to the next command